apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.ts

```diff
@@ -75,6 +78,13 @@ export function AssessmentQuestion({
         ${renderEjs(import.meta.url, "<%- include('../../partials/navbar'); %>", resLocals)}
         ${GradingConflictModal()}
         <main id="content" class="container-fluid">
+          ${AssessmentSyncErrorsAndWarnings({
+            authz_data,
+            assessment,
+            courseInstance: course_instance,
+            course,
+            urlPrefix,
+          })}
```

Comment:
This was moved to the top of the main container for consistency with other pages.